AiProgrammerApp.ts

```diff
@@ -1,18 +1,26 @@
 import {
-    IAppAccessors,
-    IConfigurationExtend,
-    ILogger,
-} from "@rocket.chat/apps-engine/definition/accessors";
-import { App } from "@rocket.chat/apps-engine/definition/App";
-import { IAppInfo } from "@rocket.chat/apps-engine/definition/metadata";
-import { SummarizeCommand } from "./commands/SummarizeCommand";
+	IAppAccessors,
+	IConfigurationExtend,
+	ILogger,
+} from '@rocket.chat/apps-engine/definition/accessors';
+import { App } from '@rocket.chat/apps-engine/definition/App';
+import { IAppInfo } from '@rocket.chat/apps-engine/definition/metadata';
+import { SummarizeCommand } from './commands/SummarizeCommand';
+import { settings } from './settings/settings';
 
 export class ThreadSummarizerApp extends App {
```

Comment:
@RyanbowZ the app class name should not be ThreadSummarizerApp, It should be related to the App name.  ~ along the lines of AIProgammer ?